integrationTests/cypress/integration/redmine_administration.feature

```diff
@@ -0,0 +1,69 @@
+Feature: Administrative Procedures
+
+  @requires_testuser
+  Scenario: cas user + no redmine user => login => create normal redmine account
+    Given the user is not member of the admin user group
+    And the user has no internal redmine account
+    And the user is logged out of the CES
+    When the user logs into the CES
+    Then the user has an internal redmine account with default privileges
+
+  @requires_testuser
+  Scenario: cas user (admin) + no redmine user => login => create admin redmine account
+    Given the user is member of the admin user group
+    And the user has no internal redmine account
+    And the user is logged out of the CES
+    When the user logs into the CES
+    Then the user has an internal redmine account with admin privileges
+
+  @requires_testuser
+  Scenario: cas user + internal default redmine user => login => preserves default account
+    Given the user is not member of the admin user group
+    And the user has an internal default redmine account
+    And the user is logged out of the CES
+    When the user logs into the CES
+    Then the user has an internal redmine account with default privileges
+
+  @requires_testuser
```

Comment:
Suggestion:
"cas user + internal cas-admin redmine user => login => demote to default redmine account" -->
"A regular CAS user, with existing admin redmine account logs into redmine and his redmine account is rightfully demoted to a regular account"